account_tax_one_vat/models/account_move_line.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, models
+
+
+class AccountMoveLine(models.Model):
+    _name = "account.move.line"
+    _inherit = ["account.move.line", "one.vat.mixin"]
+
+    @api.onchange("tax_ids")
+    def _onchange_tax_ids(self):
```

Comment:
To avoid conflicts, name the method with what it is doing, not what is triggering it
```suggestion
    def _onchange_only_one_vat_tax(self):
```